deploy/complete/terraform/variables.tf

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+# 
```

Comment:
ensure all files have the copy header

Reply:
even the YAMLs?

Reply:
![yam ls](https://user-images.githubusercontent.com/551007/80754946-9171db00-8af5-11ea-988c-fb5541a1d8a2.png)

Reply:
😆 yes, add... and I'll have some 🥔 with that